spark-3.5-scala-2.13.yaml

```diff
@@ -76,6 +76,9 @@ subpackages:
 
   - name: ${{package.name}}-compat
     description: "Compatibility package to place binaries in the location expected by upstream image"
+    dependencies:
+      provides:
+        - spark-compat
```

Comment:
is this a valid syntax in melange? should we add this ${{package.full-version}} ? like `spark-compat=${{package.full-version}}`

Reply:
it looks like its a valid syntax, nice!